website/docs/d/google_compute_instance_group_instances.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "google"
+page_title: "Google: google_compute_instance_group_instances"
+sidebar_current: "docs-google-datasource-compute-instance-group-instances"
```

Comment:
Don't forget to update this [file](https://github.com/terraform-providers/terraform-provider-google/blob/master/website/google.erb) to include your new docs page in the sidebar menu